docker/server-jetty/src/main/docker/Dockerfile

```diff
@@ -28,13 +28,26 @@ FROM $BASE
 COPY --from=install / /
 VOLUME /data
 VOLUME /cache
-HEALTHCHECK --interval=3s --retries=3 --timeout=11s CMD /opt/grpc_health_probe/grpc_health_probe -addr=localhost:10000 -connect-timeout=10s || exit 1
+HEALTHCHECK --interval=3s --retries=3 --timeout=11s CMD /bin/grpc_health_probe -addr=localhost:10000 -connect-timeout=10s || exit 1
```

Comment:
I'm confused why this change was made...

Reply:
agree